sct.py

```diff
@@ -304,8 +304,10 @@ def clean_resources(ctx, post_behavior, user, test_id, logdir, dry_run, backend)
 @click.option('--get-all-running', is_flag=True, default=False, help='All running resources')
 @sct_option('--test-id', 'test_id', help='test id to filter by')
 @click.option('--verbose', is_flag=True, default=False, help='if enable, will log progress')
+@click.option('-b', '--backend-type', type=click.Choice(["all", "aws", "gce", "eks", "docker", "azure"]),
```

Comment:
Those are not all the options available for backend, the list is defined in sct_config.py

Reply:
fixed